meshcapade_addon/blender.py

```diff
@@ -1,11 +1,56 @@
 import bpy
 from .globals import (
-    FBX_TYPE,
-    OBJ_TYPE,
+    SUPR_JOINT_NAMES,
+    SMPLH_JOINT_NAMES,
+    SMPLX_JOINT_NAMES,
 )
 from mathutils import Vector, Quaternion
+from math import radians
+import os
+
+def get_joint_names(SMPL_version):
+    if (SMPL_version == None):
```

Comment:
ok i changed it so that if it doesn't detect a SMPL version, it defaults to SMPLX, which is going to work better than just sending "error" to the SMPL version which won't ever work.  

I also changed all of the if statements in the whole project that had parenthesized single conditions